extension/docs/DEV.md

```diff
@@ -36,16 +37,27 @@ cp .env.example .env.development
 cp .env.example .env.production
 ```
 
-Build Bergamot Translator for WASM:
+## Building Bergamot Translator WASM resources and importing them to the extension
+
+1. Make sure all submodules are initiated:
+
+```bash
+git submodule update --init --recursive
+```
+
+2. Put the model files to be included in the build into `../bergamot-translator/models/` according to the instructions [here](../bergamot-translator/README.md).
```

Comment:
Right now, if you do this the builds will fail. We need to change instructions in `bergamot-translator` repo appropriately.

However, it will save you, me and user of this repo a lot of pain if you just point user to `bergamot-translator/README.md` for instructions on how to package models instead of writing the name of the folders explicitly here.

Reply:
Will remove the explicit path